databricks/koalas/utils.py

```diff
@@ -376,12 +385,11 @@ def align_diff_frames(
     return kdf
 
 
-def align_diff_series(func, this_series, *args, how="full"):
-    from databricks.koalas.base import IndexOpsMixin
+def align_diff_series(func, this_series: "Series", *args, how: str = "full") -> "Series":
```

Comment:
I'm just wondering whether we should keep `align_diff_series` here or move to `series.py` since it is only for Series.

Reply:
Yeah, maybe we can move it, but I'd leave it to the future PRs.

Reply:
Sounds good!